src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1813,6 +1813,32 @@ def update_resource(cmd, parameters, resource_ids=None,
          for id_dict in parsed_ids])
 
 
+def patch_resource(cmd, properties=None, resource_ids=None, resource_group_name=None,
+                   resource_provider_namespace=None, parent_resource_path=None, resource_type=None,
+                   resource_name=None, api_version=None, latest_include_preview=False, is_full_object=False):
```

Comment:
```suggestion
def patch_resource(cmd, properties, resource_ids=None, resource_group_name=None,
                   resource_provider_namespace=None, parent_resource_path=None, resource_type=None,
                   resource_name=None, api_version=None, latest_include_preview=False, is_full_object=False):
```
Do we need to make the `--properties` required, right?